.storybook/main.ts

```diff
@@ -68,6 +69,8 @@ const config: StorybookConfig = {
         return !(isStyledSystemProp || isHTMLElementProp)
       },
     },
+
+    reactDocgen: "react-docgen-typescript"
```

Comment:
I don't see this dep installed. Is it missing or is it a transitive dep?

Reply:
I guess it's transitive? I'm not familiar with the term.

This is a part of the main storybook. It needs to be set in v8 because it was the default in v7, but they switched to `react-docgen` as the default. The one set here is needed to filter out all those extraneous Chakra props.

Reply:
Ah I see, ok. Yes, it is installed by sb 👍🏼